Documentation/ViewDesigner/CommonSteps.md

```diff
@@ -0,0 +1,125 @@
+**Original:** _search * | summarize count() by Type_
+
+**Updated:** _search * **| where TimeGenerated {TimeRange}** | summarize count() by Type_
+
+## Including a List
+Most of the View Designer views include a list, this standard list can be re-produced utilizing the following steps.
+
+![Tile List](./TileList.png)
+
+You will need to add a visualization by clicking **Add query** from the cell options
+
+![Add Param](./AddParam.png)
+
+View Designer employs a default query that matches the syntax from the Original example. This can be updated by changing the query to the updated form.
+
+**Original:** _search * | summarize **AggregatedValue** = count() by Type_\
+**Updated:** _search * | summarize **Count** = count() by Type_
+
+This will generate a list that looks similar to the following:
+
+![List Example](./ListEx.png)
+
+## Enabling Sparklines
+A common feature for grids is to add sparklines to summarize various data patterns over time. View Designer offers the Enable Sparklines feature for all lists, as does Workbooks.
+
+To include Sparklines in your data that match View Designer, you will need to join the data with your original query. You will need to add the following line
+
+_| join kind = inner (search * \
+| make-series Trend = count() default=0 on TimeGenerated from \{TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by Type)\
+on Type_
+
+Then under **Column Settings**
+![Column Settings](./ColumnSettings.png)
+
+Update the **Column renderer** dropdown to be a Spark area
+![Sparklines](./Sparkline.png)
+
+Then save the settings and re-running the query will now update your table to include a Sparkline
+
+Example:\
+**Original:**\
+_search *\
+| summarize AggregatedValue = count() by Type) on Type_\
+**Updated:**\
+_search *\
+| summarize AggregatedValue = count() by Type\
+**| join kind = inner (search * \
+| make-series Trend = count() default=0 on TimeGenerated from \{TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by Type\
+) on Type\
+| project Type, AggregatedValue, Trend**_\
+
+Resulting grid will look similar to below:
+![Sparkline Example](./SparkEx.png)
+
+## Advanced Cell Settings
+To mirror View Designer, some common steps such as changing the size of Workbook cells or adding pins and external links to logs might be utilized.
+
+To access Advanced Settings select the gear icon at the bottom of each cell
+
+![Advanced Settings](./AdvSet.png)
+
+This will bring up a menu with various options
+
+![Advanced Settings Settings](./AdvSetSettings.png)
+
+To add a pin and a link to an external query simply select the corresponding checkboxes.
+
+To add a title to your cell, simply type the desired title into the **Chart title** section.
+
+By default any Workbooks cell is set to take up the entire page width, but users can adjust this by scaling the cell down, by clicking under the **Style** tab of the Advanced Settings menu
+
+![Advanced Settings Style](./AdvSetStyle.png)
+
+ 
+## Additional Parameters
+

```

Comment:
Seems we miss telling user how to use the parameter though?
We can link to existing documentation also: https://github.com/microsoft/Application-Insights-Workbooks/tree/master/Documentation/Parameters